### DISCUSSION TITLE
Support to multiple workspace per user

### DISCUSSION THREAD

**Discussion by 07Gond** [outline/outline#1411] [General]
I want to try this in my company.
But we have different workspaces with same email and same slack workspace.
I was thinking for example, multiple subdomains with same user.
I want to ask if the support for multiple workspaces with same user(As Notion or Slack do) is in the near roadmap.

**Comment by 07Gond** [comment]
Any idea here?

**Reply by tommoor** [reply]
This is supported in the same manner as Slack – there is a single user per workspace, you can be signed into multiple workspaces at once on different subdomains.

**Reply by 07Gond** [reply]
If I have one slack workspace, one Gapp domain and one Outline team and then go and try to create another team I am redirected to my previous team because is asking me to login with some of those options.

**Reply by tommoor** [reply]
@07Gond what kind of usecases are you thinking of? We're evaluating have a layer of navigation hierarchy above collections probably called "workspaces"... or are you talking more about completely separate teams with the same login?

**Reply by 07Gond** [reply]
Hi @tommoor 
I was looking for a way to have more than 1 workspace per user as is working for example in Notion.
In that way we could have separation of concerns, clients, etc...
In my Company we have different clients and some information is behind NDAs, that's why I was looking for something like this.
